src/l500/l500-color.cpp

```diff
@@ -566,7 +566,14 @@ namespace librealsense
     {
         std::vector<tagged_profile> tags;
 
+        auto usb_spec = get_usb_spec();
+        bool usb3mode = (usb_spec >= platform::usb3_type || usb_spec == platform::usb_undefined);
+
+        uint32_t width = usb3mode ? 1280 : 960;
+        uint32_t height = usb3mode ? 720 : 540;
+
         tags.push_back({ RS2_STREAM_COLOR, -1, 1280, 720, RS2_FORMAT_RGB8, 30, profile_tag::PROFILE_TAG_SUPERSET | profile_tag::PROFILE_TAG_DEFAULT });
```

Comment:
Shouldn't this be using `width` & `height`?

Reply:
Correct - Fixed.
It passed my test cause on USB2 the color sensor only have 1 resolution.